pages/handbook/product-engineering/how-we-work/code-review.mdx

```diff
@@ -0,0 +1,35 @@
+# Code Review
+
+## When a PR author should ask for review
+- Low risk (2 way door): go ahead and merge. Engineer is responsible for monitoring Datadog in case something goes wrong. Notify on-call engineer for roll-backs if required.
+  - Product expert review: sometimes we build a larger change in someone else’s area of responsibility. Feel free to ask for feedback from [DRI](https://docs.google.com/spreadsheets/d/1gOvWf_uSAtcXxWkR_gMuWX8-OYmSJNIPXTmPGfZ2DGY/edit?gid=0#gid=0).
+- Risk (1 way door): Assign Max as reviewer (Max sees open PR reviews in Linear, SLA 24h, ping if more urgent). **Non exhaustive list of PRs which require a review**: database migrations, changes in the public API, changes in SDK signatures, auth changes, major changes in the ingestion pipeline, larger infra changes. If you are unsure, ask Max for a review.
+- New joiners should get all their PRs reviewed for the first 1 month at least. This helps a lot with knowledge sharing and making sure we use abstractions within our code base the right way. Afterwards, we move towards no-review merges.
```

Comment:
Can you highlight this in a info box?

Also "explain the why" (see my task relevant maturity comment); Not because we don't trust them, but because we believe that to become an 10x engineer in our codebase one needs to understand the context around domain, architecture, conventions, where risks lie, speed vs quality tradeoff etc